### ISSUE TITLE
Bundling errors with React 0.14 

### ISSUE THREAD

**Issue by insin** [react-bootstrap/react-bootstrap#1393]
A bunch of internals have been moved to the [fbjs](https://github.com/facebook/fbjs) package:

```
ERROR in ./~/react-bootstrap/lib/OverlayTrigger.js
Module not found: Error: Cannot resolve module 'react/lib/warning'
 @ ./~/react-bootstrap/lib/OverlayTrigger.js 32:23-51

ERROR in ./~/react-bootstrap/lib/utils/deprecationWarning.js
Module not found: Error: Cannot resolve module 'react/lib/warning'
 @ ./~/react-bootstrap/lib/utils/deprecationWarning.js 11:23-51
```

**Comment by jquense** [comment]
Yeah 0.26.0 isn't really compatible, 0.27.0 coming shortly

**Comment by taion** [comment]
Fixed by #1394.
